x-pack/plugins/cases/server/common/limiter_checker/index.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import Boom from '@hapi/boom';
+
+import type { CommentRequest } from '../../../common/api';
+import type { AttachmentService } from '../../services';
+import type { Limiter } from './types';
+import { AlertLimiter } from './limiters/alerts';
+import { FileLimiter } from './limiters/files';
+
+export class LimitChecker {
```

Comment:
Could you please add unit tests for all classes in the `limit_checker` folder? Also as this `LimitChecker` is specific to comments maybe we can rename it to `AttachmentsLimitChecker` to be clearer in the code.